README.md

```diff
@@ -1,5 +1,128 @@
+
+* Download the [zip file](https://github.com/systems-cs-pub-ro/quiz-manager-moodle/zipball/master)
+* Unzip the file in the correct location: `/your/moodle/path/question/type`
+* Rename the directory to `quizmanager`
+
+### Manual Installation Using Git
+
+To install using git, type this command:
+```
+git clone https://github.com/systems-cs-pub-ro/quiz-manager-moodle.git /your/moodle/path/question/type/quizmanager
+```
+
+
+# Adding Questions
+
+* After creating a quiz, access it and press the `Edit quiz` button
+* On the right side of the page, go to `Add > + a new question` and select the `quizmanager` question type
+* Select the category where you want your question to be from
+* Set the difficulty from the given list _(Easy - Hard)_
+* Set the topic that you want your question to have (Note: topics are case **insensitive**)
+* Press the `Save changes` button
+* Repeat the instructions for all the questions you want in your quiz
+
+# Managing Questions
+
+The plugin can use any question types existent in the question bank, as long as the have the associated difficulty and topic tag.
+
+If imported from an external database, the following rules must be followed:
+* The difficulty tag must be one from the following list: `Easy, Easy-Medium, Medium, Medium-Hard, Hard`
+* The topic tag can take any form, as long as it is consistent with the set topic in the question edit form
+(see [Adding Questions](/README.md/#adding-questions))
+* The `last_used` tag is optional (if not present will be considered 0), and the format is: `last_used:time`, where
+time is an integer representing time since the Epoch (00:00:00 UTC, January 1, 1970), measured in seconds
+
+Details for the importing procedure can be found in the [Moodle documentation](https://docs.moodle.org/311/en/Moodle_XML_format)
+
+The following example can be used for importing questions with tags in MXML format (note that you can modify
+any detail about the questions, the example only shows the use of tags in MXML):
+```xml
+<?xml version="1.0" ?>
+
+<quiz>
+    <question type="multichoice">
+        <name>
+            <text>
+                Question Name Goes Here
+            </text>
+        </name>
+        <questiontext format="html">
+            <text>
+		    Question Text Goes Here
+            </text>
+        </questiontext>
+
+        <tags>
+            <tag>
+                <text>
+			First Tag Goes Here
+                </text>
+            </tag>
+            <tag>
+                <text>
+			Second Tag Goes Here
+                </text>
+            </tag>
+	    <tag>
+		<text>
+			Third Tag Goes Here
+		</text>
+	    </tag>
+        </tags>
+
+        <answer fraction="100">
+            <text>
+                Correct Answer Goes Here
+            </text>
+        </answer>
+
+        <answer fraction="0">
+            <text>
+                Incorrect Answer Goes Here
+            </text>
+        </answer>
+    </question>
+</quiz>
+```
+
+## __User stories__:
```

Comment:
I would remove the user stories from the `README.md` and place it another file: `UserStories.md` or `user_stories.md`, possibly inside a `doc/` subfolder.